### PR TITLE
Document referencing bugs when syncing release

### PR DESCRIPTION
**Pull request by lbarcziova**
Related to packit/packit#1920

### CODE REVIEW

File: docs/configuration/actions.md [hunk 1]
```diff
@@ -204,6 +204,8 @@ There are also action-specific environment variables, which you can see below.
 
 * `PACKIT_PROJECT_VERSION` — version to be set in the specfile, set when relevant 
 (e.g. when syncing upstream release downstream)
+* `PACKIT_RESOLVED_BUGS` - bugs resolved by the release separated by space, set 
+when syncing upstream release downstream
```
Comment (lachmanfrantisek) [inline docs/configuration/actions.md:208]: Could you please either mention the syntax or add an example?

File: docs/configuration/downstream/pull_from_upstream.md [hunk 2]
```diff
@@ -7,7 +7,10 @@ sidebar_position: 1
 
 A dist-git only job that opens a new dist-git pull request in
 src.fedoraproject.org when a new upstream release happens using a notification
-from [release-monitoring.org](https://release-monitoring.org/).
+from [release-monitoring.org](https://release-monitoring.org/). Upstream Release Monitoring creates a Bugzilla
+for the new upstream release and this Bugzilla is by default referenced in the dist-git changelog as
+`- Resolves rhbz#xz` or in the commit message in case of using `%autochangelog`. This behaviour can be customized
```
Comment (lachmanfrantisek) [inline docs/configuration/downstream/pull_from_upstream.md:12]: Actually, don't we add this to the commit message even if not using `%autochangelog`?

File: docs/configuration/downstream/pull_from_upstream.md [hunk 3]
```diff
@@ -54,6 +57,14 @@ This will take the Packit configuration file from the default branch of the dist
 
     /packit pull-from-upstream --with-pr-config
 
+
+`pull-from-upstream` triggered by the Upstream Release Monitoring automatically handles the Bugzilla created by Upstream
+Release Monitoring (by default adds `Resolves` to changelog/commit and exposes `PACKIT_RESOLVED_BUGS` to the `changelog-entry` and `commit-message`
```
Comment (lachmanfrantisek) [inline docs/configuration/downstream/pull_from_upstream.md:62]:
```suggestion
Release Monitoring (by default adds `Resolves` to changelog/commit and exposes `PACKIT_RESOLVED_BUGS` environment variable to the `changelog-entry` and `commit-message`
```

File: docs/configuration/downstream/pull_from_upstream.md [hunk 4]
```diff
@@ -54,6 +57,14 @@ This will take the Packit configuration file from the default branch of the dist
...
+actions). In case of retriggering, you need to specify the bugs like this:
```
Comment (lachmanfrantisek) [inline docs/configuration/downstream/pull_from_upstream.md:63]: Do you think we should/can suggest a way to find the Bugzilla?

**Review by lachmanfrantisek — COMMENTED** [review]
Thanks!

File: docs/configuration/downstream/pull_from_upstream.md (hunk 4 above)
Comment (lbarcziova) [reply docs/configuration/downstream/pull_from_upstream.md:63]: good point!

File: docs/configuration/downstream/pull_from_upstream.md (hunk 2 above)
Comment (lbarcziova) [reply docs/configuration/downstream/pull_from_upstream.md:12]: no, we check whether the specfile uses %autochangelog... but we could still change this before releasing, WDYT

Comment (lbarcziova) [reply docs/configuration/downstream/pull_from_upstream.md]: after discussing with Franta, the behaviour was changed and therefore the docs updated as well

File: docs/configuration/downstream/pull_from_upstream.md (hunk 4 above)
Comment (lachmanfrantisek) [reply docs/configuration/downstream/pull_from_upstream.md:63]: Thanks!

**Review by lachmanfrantisek — APPROVED** [review]
:pray:

**Pull request merged**